src/DataCollection.Shared/ViewModels/FeatureViewModel.cs

```diff
@@ -50,17 +50,17 @@ public PopupManager PopupManager
             get { return _popupManager; }
             set
             {
-                if (_popupManager != value)
+                if (_popupManager != value && value != null)
```

Comment:
This makes the `value != null` check below redundant -- can that one be removed?